packages/eslint-plugin-turbo/lib/rules/no-undeclared-env-vars.ts

```diff
@@ -46,9 +62,11 @@ function create(context: Rule.RuleContext): Rule.RuleListener {
       console.error(`Unable to convert "${allowed}" to regex`);
     }
   });
+
+  const cwd = normalizeCwd(context.getCwd ? context.getCwd() : undefined);
```

Comment:
```suggestion
  const cwd = normalizeCwd(context?.getCwd());
```

Reply:
That's not going to do what you're thinking it will: `window?.foo()` => `(undefined)()` => `TypeError`.